File: src/main/java/hxy/server/socket/entity/SslInfo.java
package hxy.server.socket.entity;

/**
 * @ClassName SslInfo
 * @Author hxy
 * @Date 2020/4/8 18:38
 */
public class SslInfo {
    private boolean open = false;
    private String certFilePath = null;
    private String keyFilePath = null;

    public SslInfo() {
    }

    public boolean isOpen() {
        return open;
    }

    public void setOpen(boolean open) {
        this.open = open;
    }

    public String getCertFilePath() {
        return certFilePath;
    }

    public void setCertFilePath(String certFilePath) {
        this.certFilePath = certFilePath;
    }

    public String getKeyFilePath() {
        return keyFilePath;
    }

    public void setKeyFilePath(String keyFilePath) {
        this.keyFilePath = keyFilePath;
    }
}

File: server/src/main/kotlin/nafos/server/SslInfo.kt
package nafos.server

/**
 * @Description Ssl相关信息
 * @Author      xinyu.huang
 * @Time        2019/11/24 1:24
 */
class SslInfo(
        var open: Boolean = false,

        var certFilePath: String? = null,

        var keyFilePath: String? = null
)